x-pack/solutions/security/test/scout/ui/tests/security_navigation.spec.ts

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { expect, tags, test } from '@kbn/scout-security';
+
+test.describe('Security Solution Navigation', { tag: tags.ESS_ONLY }, () => {
+  test.beforeEach(async ({ browserAuth, pageObjects }) => {
+    await browserAuth.loginAsAdmin();
+    await pageObjects.securityNavigation.goToTimelines();
+  });
+
+  test('navigates to the Dashboards landing page', async ({ page, pageObjects }) => {
+    await pageObjects.securityNavigation.clickDashboards();
+    expect(page.url()).toContain('/app/security/dashboards');
+  });
+
+  test('navigates to the Overview page', async ({ page, pageObjects }) => {
+    await pageObjects.securityNavigation.clickOverview();
+    expect(page.url()).toContain('/app/security/overview');
+  });
```

Comment:
does it really needs to be a new `test` block? For each `test` block scout creates a fresh browser context, which takes quite some time if you have 10x blocks. The alternative we suggest is to group navigation checks based on levels and use `test.step` to have a nice clear report:

```
  test.step('navigates to the Dashboards landing page', async ({ page, pageObjects }) => {
    await pageObjects.securityNavigation.clickDashboards();
    expect(page.url()).toContain('/app/security/dashboards');
  });

  test.step('navigates to the Overview page', async ({ page, pageObjects }) => {
    await pageObjects.securityNavigation.clickOverview();
    expect(page.url()).toContain('/app/security/overview');
  });

....
```